readme-summaries.txt

```diff
@@ -12,4 +12,7 @@ Iteration 3
 We added functionality to allow a user to create a new cover. To do so, we first created a function, collectUserInput, to save a user's input when they add new covers, titles, and descriptors to the form on the Make Your Own Cover page. We then created a function, createNewCover, that generates a new instance of the Cover class with the user's input and displays it on the main page so that when a user returns Home, they see that cover. We also refactored some functions to make them more dynamic and fixed a bug in the titles array that was causing some titles to display as 'undefined'.
 
 Iteration 4
-Next, we added functionality so that when the 'Save Cover' button is clicked, the saveCurrentCover function is exectuted, which tests if that current cover matches one already in the savedCovers array, and if not, adds it to the array. Next, we expanded upon what happens when a user clicks 'View Saved Covers' by calling a new function, displayMiniCovers, at the end of our displaySavedView function. This new function replaces the innerHTML of the saved covers section with blocks of code for each cover, using string interpolation to insert its id, image, title, and descriptors. To get the covers to display correctly on the page, we used the insertAdjacentHTML() method and styled each block using the mini-cover CSS class.  
+Next, we added functionality so that when the 'Save Cover' button is clicked, the saveCurrentCover function is exectuted, which tests if that current cover matches one already in the savedCovers array, and if not, adds it to the array. Next, we expanded upon what happens when a user clicks 'View Saved Covers' by calling a new function, displayMiniCovers, at the end of our displaySavedView function. This new function replaces the innerHTML of the saved covers section with blocks of code for each cover, using string interpolation to insert its id, image, title, and descriptors. To get the covers to display correctly on the page, we used the insertAdjacentHTML() method and styled each block using the mini-cover CSS class.
+
+Iteration 5
+We added functionality so that when a user double-clicks a cover from the Saved Covers view, it is deleted. We did so by creating an event listener for the savedCoversSection, which fires a function, deleteSelectedCover, when a cover is double-clicked. The deleteSelectedCover function accesses the id associated with the specific element selected, and uses that to remove the cover with the corresponding id from the savedCovers array. Then, the displayMiniCovers function is called again to re-render the saved Covers view with just the remaining elements in the savedCovers array.
```

Comment:
Thanks for getting this in so quickly. Thanks for driving while we did the project review!  MIGHT get README update for iter 4 pushed tonight still?!?  GOOD WORK TODAY!